### PR TITLE
Return exit code: 0 from cli on 'up' where there are no migrations

### PR DESCRIPTION
**Pull request by river-jade**

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -165,6 +165,7 @@ export default class Migrator {
         this.log('There are no migrations to run'.yellow);
         this.log(`Current Migrations' Statuses: `);
         await this.list();
+        return []
```
Comment (balmasi) [inline src/lib.js]: I think you want this in the `else` block @river-jade 
return [] if it's NOT CLI

Comment (river-jade) [inline src/lib.js]: i'm not sure what you mean. this seems right to me and we've been using it in production. 

the intention is to not error in the case that there are no migrations and we're running in command line mode, right? what am i missing? thanks!

Comment (balmasi) [inline src/lib.js]: Ah I see what's going on. I was referring to another bug I noticed lol where in API mode (as npm package) it was throwing an error as well, when it shouldn't in either case.

Thanks! @river-jade  I won't merge this so I can fix both issues~! <3

Comment (river-jade) [inline src/lib.js]: ah ok, got it. yes, i wondered if that was intended behaviour. i figured the Error had been put in for a reason. so you want to return [] for both cases? if so, i'll update the PR.

**Comment by sasos90** [comment]
When do you intend to merge this pull request? Thanks!

**Comment by menocomp** [comment]
@balmasi  Thanks for maintaining this nice library.
Any plans to merge this PR?

**Comment by balmasi** [comment]
Apologize for taking so long. I didn't have access to my github account.

I've just released the fix to this in 4.0.0. It's different than the solution above so I'm closing this one.

Thanks for making a PR!

**Pull request closed without merging**